docs/pages/get-started/create-a-project.mdx

```diff
@@ -1,117 +1,22 @@
 ---
 title: Create a project
-description: Learn how to create a new Expo project and run it on your device.
+description: Learn how to create a new Expo project.
+hideTOC: true
 ---
 
-import { Collapsible } from '~/ui/components/Collapsible';
 import { Terminal } from '~/ui/components/Snippet';
-import ImageSpotlight from '~/components/plugins/ImageSpotlight';
-import { Step } from '~/ui/components/Step';
-import { BoxLink } from '~/ui/components/BoxLink';
-import { BookOpen02Icon, GraduationHat02Icon } from '@expo/styleguide-icons';
 
-The process of running a new Expo project consists of three steps. You start by initializing a new project, then start the development server with Expo CLI and finally open the app on your device with Expo Go to see your changes live.
+System requirements:
 
-<Step label="1">
+- [Node.js (LTS)](https://nodejs.org/en/).
+- macOS, Windows (Powershell and WSL 2), and Linux are supported.
 
-## Initialize a new project
+We recommend creating a project with the default project template, which includes example code to help get you started. To create a new project, run the following command:
```

Comment:
```suggestion
We recommend creating a project with `create-expo-app` that sets up a default project template and includes example code to help get you started. To create a new project, run the following command:
```

- Adding a little bit of context about what `create-expo-app` helps telling what it does (which is the context already added here).

Reply:
I am including this comment with the following:

```
We recommend starting with the template project created by `create-expo-app`. This project includes example code to help you get started.

To create a new project, run the following command:
```

Again, and with all my rewordings, please lmk if you prefer something else!

Reply:
This is perfect! Thanks!